java/daikon/DynComp.java

```diff
@@ -181,13 +205,14 @@ public static void check_args(Options options, String[] target_args) {
   }
 
   /**
-   * Starts the target program with the Java agent setup to do the transforms. All Java agent
+   * Starts the target program with the Java agent set up to do the transforms. All Java agent
    * arguments are passed to it. Our classpath is passed to the new JVM.
```

Comment:
Who is "Our"?  (It could by DynComp, or the program being instrumented.)

Reply:
Changed to "The current classpath is passed ..."